prov/rxd/src/rxd_cq.c

```diff
@@ -411,6 +411,9 @@ static void rxd_handle_rts(struct rxd_ep *ep, struct rxd_pkt_entry *pkt_entry)
 	struct rxd_rts_pkt *pkt = (struct rxd_rts_pkt *) (pkt_entry->pkt);
 	int ret;
 
+	if (pkt->base_hdr.version != RXD_PROTOCOL_VERSION)
+		return;
```

Comment:
I would at least add warning messages here and below to indicate a version mismatch occurred.

Reply:
Will do